python/tvm/relay/op/contrib/dnnl.py

```diff
@@ -224,22 +426,56 @@ def partition_for_dnnl(mod, params=None):
 
     if params:
         mod["main"] = bind_params_by_name(mod["main"], params)
-    seq = tvm.transform.Sequential(
+    from tvm.relay.testing.temp_op_attr import TempOpAttr
+
+    with TempOpAttr("nn.conv2d", "FTVMLegalize", legalize_group_conv):
+        with TempOpAttr("nn.conv2d_transpose", "FTVMLegalize", legalize_group_conv_transpose):
+            seq = tvm.transform.Sequential(
+                [
+                    transform.CanonicalizeOps(),
+                    transform.InferType(),
+                    transform.SimplifyInference(),
+                    transform.FoldConstant(),
+                    transform.FoldScaleAxis(),
+                    # fold consecutive add ops to simplify pattern `conv2d-bias_add-bn-relu`
+                    transform.SimplifyExpr(),
+                    transform.FoldConstant(),
+                    # alter group conv /conv_transpose layout to `GOIHW` / `GIOHW`
+                    transform.Legalize(),
+                    transform.FoldConstant(),
```

Comment:
Is there any dependency in this order list?

Reply:
Yes, these pass should be executed in this order.